easybuild/easyconfigs/a/ALPS/ALPS-2.3.0-foss-2016b-Boost-1.63-Python-2.7.12.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+separate_build_dir = True
+
+name = 'ALPS'
+version = '2.3.0'
+cxxstd = 'c++03'
+
+homepage = "http://alps.comp-phys.org/"
+description = "ALPS project"
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': cxxstd}
+
+
+sources = ['%(namelower)s-%(version)s-src.tar.gz']
+source_urls = ['http://alps.comp-phys.org/static/software/releases/']
+
+patches = [
+    #'ALPS-find-python.patch'
+]
+
+boostver = '1.63.0'
+pyver = '2.7.12'
+
+dependencies = [
+    ('CMake', '3.6.2'),
+    ('Boost', boostver, '-Python-%s' % (pyver)),
+    ('HDF5', '1.8.17')
+]
+versionsuffix='-Python-%s' % (pyver,)
```

Comment:
please move this line right below `version`, and use the `%(pyver)s` template (also, spaces around `=` please)

```python
versionsuffix = '-Python-%(pyver)s`
...
dependencies = [
    ('Python', '2.7.12'),
    ...
]